docs/source/config/inputs/http.rst

```diff
@@ -80,6 +75,11 @@ Config:
     Severity level of errors, unreachable connections, and non-200 responses
     of successful HTTP requests. Defaults to 1 (alert).
 
+    Subsection defining headers for the request. By default the User-Agent
+    header is set to "Heka"
+- body (string):
+    .. versionadded:: 0.5
+
```

Comment:
Looks like you were trying to move the `headers`  piece down here to the bottom, but you instead moved the headers description and the following `body` label.